src/http/cookie.cr

```diff
@@ -104,19 +144,44 @@ module HTTP
       CookieString    = /(?:^|; )#{Regex::CookiePair}/
       SetCookieString = /^#{Regex::CookiePair}(?:;\s*#{Regex::CookieAV})*$/
 
-      def parse_cookies(header)
+      # Yeilds an `HTTP::Cookie` for each cookie in the header.
```

Comment:
```suggestion
      # Yields an `HTTP::Cookie` for each cookie in the header.
```
The entire `Parser` module is :nodoc:, so this doesn't need extensive documentation. Doesn't hurt, though.